### ISSUE TITLE
Shoulder internal rotation limited

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10998]
**Preferred term label:**
Shoulder internal rotation limited

**Synonyms**

**Definition (free text, please give PubMed ID)**
Difficulty rotating the lower part of the arm towards the body (tested with the elbow flexed and the elbow held next to the side of the body or shoulder 90 degrees abducted to the side) secondary to pain, weakness, or mechanical block; reduced shoulder range of motion; stiff shoulder

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Shoulder internal impingement syndrome, Pectoralis major rupture, Proximal humeral epiphysiolysis, Rotator cuff tear arthropathy

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001170
- label: Limited shoulder internal rotation
- parents: [Limited shoulder movement (HP:0006467)](https://hpo.jax.org/app/browse/term/HP:0006467)
- pmid: 
- def: Difficulty rotating the lower part of the arm towards the body (tested with the elbow flexed and the elbow held next to the side of the body or shoulder 90 degrees abducted to the side) secondary to pain, weakness, or mechanical block.
- comment: Conditions characterized by this finding include shoulder internal impingement syndrome, pectoralis major rupture, proximal humeral epiphysiolysis, and rotator cuff tear arthropathy.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10998
- synonyms: Shoulder internal rotation limited(NONE)
